### ISSUE TITLE
Screenreader Issues

### ISSUE THREAD

**Issue by hahnln** [CenterOnSocietyAndHealth/vdh-opioid-2025#28]
@marc-redmond I'm still learning NVDA, but it's actually [the second-most popular screenreader tech](https://webaim.org/projects/screenreadersurvey10/#:~:text=NVDA%20is%20again%20the%20most,respondents%20outpacing%20JAWS%20at%2060.5%25.). Do you have access to it? That may be the fastest way to pinpoint issues for folks who use it. (Long story, but I haven't been able to test with JAWS, as the library actually doesn't have it as advertised.)

If not, the IT support group at Cabell Library may be able to give you a loaner laptop with it installed. Let me know if you need help getting one, I could maybe bring it to your house if helpful!

A few issues:

- [x] need more landmarks on the home page (see [Stark labels in Figma](https://www.figma.com/design/OxBKNFrosS3ojbEvb8MVE3/Stark-Annotations?node-id=2371-2311&t=LYSYmWOAgGtkGnyO-1))

- [x] The instructions for the dropdown selector for the map feel a little unclear. It says the dropdown is editable but it could be clearer I think.

- [x] The map feels like a keyboard trap right now.

- [ ] The text is super-choppy in some unexpected places, where I don't see any kind of variables or bolding, italics, anything like that. I haven't been able to upload the screenshots I took but will add these Monday.
